tools/execsnoop.py

```diff
@@ -272,7 +275,9 @@ def check_cpu_filed():
     print("%-8s" % ("TIME(s)"), end="")
 if args.print_uid:
     print("%-6s" % ("UID"), end="")
-print("%-16s %-7s %-7s %-4s %3s %s" % ("PCOMM", "PID", "PPID", "CPU", "RET", "ARGS"))
+if args.print_cpu:
+    print("%-6s" % ("CPU"), end="")
+print("%-16s %-7s %-7s %3s %s" % ("PCOMM", "PID", "PPID", "RET", "ARGS"))
```

Comment:
I would like CPU column after 'PPID' and before 'RET' and this confirms to most other bcc tools where PCOMM/PID/PPID is displayed earlier in columns.

Reply:
Thanks reviewing, I rearranged the CPU column display position. Now it's after 'PPID' and before 'RET'.